Makefile

```diff
@@ -8,6 +8,9 @@ FLAGS=-X main.Version=$(VERSION) -s -w
 CERT="Developer ID Application: 99designs Inc (NRM9HVJ62Z)"
 SRC=$(shell find . -name '*.go')
 
+test:
+	go test $(shell go list ./... | grep -v /vendor/)
```

Comment:
Which would mean we'd need to install glide on Travis. Because it has vendors committed glide is just needed for updating.